src/hooks/use-site-sync-management.ts

```diff
@@ -0,0 +1,92 @@
+import { useState, useEffect, useCallback } from 'react';
+import { getIpcApi } from '../lib/get-ipc-api';
+import { useAuth } from './use-auth';
+import { SyncSite, useFetchWpComSites } from './use-fetch-wpcom-sites';
+import { useSiteDetails } from './use-site-details';
+
+const useSiteSyncManagement = () => {
+	const [ connectedSites, setConnectedSites ] = useState< SyncSite[] >( [] );
+	const { isAuthenticated } = useAuth();
+	const { syncSites } = useFetchWpComSites( connectedSites );
```

Comment:
As mentioned before, let's keep this fetch and return `syncSites` and `isFetching` from this hook.

Reply:
Updated in https://github.com/Automattic/studio/pull/610/commits/91efc5acbb687e0e33a892b75764508e13549f13